modules/hcp-ecs-client/services.tf

```
@@ -33,10 +33,12 @@ resource "aws_iam_role" "frontend-task-role" {
       }
     ]
   })
+
+  name_prefix = local.secret_prefix
```

```suggestion
  name_prefix = local.scope
```

I would like to use `scope` for these things. `secret_prefix` should only be used for secret prefixes for the acl controller.